src/client/delphi_epidata.js

```diff
@@ -573,10 +573,15 @@ function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _d
             'signals': signals,
             'time_type': time_type,
             'geo_type': geo_type,
-            'time_values': _list(time_values),
-            'geo_value': geo_value
+            'time_values': _list(time_values)
           };
 
+          if (Array.isArray(geo_value)) {
+            params.geo_values = geo_value.join(',');
+          } else {
+            params.geo_value = geo_value;
+          }
+
```

Comment:
Would `_list()` be able to handle this, and save some duplication?

Reply:
in the current version changing it won't change much, since the array and plain version needs another parameter name `geo_value` vs `geo_values`. Thus, the `if` will remain.